web-components/package.json

```diff
@@ -64,7 +68,7 @@
     "react-helmet": "^6.0.0",
     "react-image-crop": "^9.0.5",
     "react-lazyload": "^3.2.0",
-    "react-map-gl": "5.3.21",
+    "react-map-gl": "^7.1.7",
```

Comment:
upgraded our react-map-gl version so there are a few changes below that can be found in the existing `StaticMap` component that are not necessarily related to the initial purpose of this work